src/NuProj.Package.Dev14/NuProj.Package.Dev14.csproj

```diff
@@ -17,7 +17,7 @@
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>NuProj.ProjectSystem</RootNamespace>
-    <AssemblyName>NuProj.ProjectSystem</AssemblyName>
+    <AssemblyName>NuProj.ProjectSystem14</AssemblyName>
```

Comment:
AFAIK VS/MSBuild is putting the version number after a `.`. I should probably follow that.